src/views/HrReports/BloodReports/BloodReports .jsx

```diff
@@ -22,15 +29,7 @@ const BloodReports = () => {
         return state.getEmployeeBloodgrp.empBlood
     })
```

Comment:
return state.getEmployeeBloodgrp.empBlood 

Sometimes this will give 'undefined' to avoid this 
use like this
return state.getEmployeeBloodgrp.empBlood  ?? 0

Nullish coalescing operator (??)